egs2/chime7_task1/asr1/local/gss_micrank.py

```diff
@@ -0,0 +1,210 @@
+import argparse
+import lhotse
+import torchaudio
+import torch
+from copy import deepcopy
+import tqdm
+from pathlib import Path
+import os
+from torch.utils.data import DataLoader, Dataset
+
+
+class EnvelopeVariance(torch.nn.Module):
```

Comment:
Channel selection is based on Envelope variance right now. 
It is not guarantee it will work because of overlapped speech